Apps/Examples/Examples/All Examples/SelectAnnotationExample.swift

```diff
@@ -92,6 +92,8 @@ public class SelectAnnotationExample: UIViewController, ExampleProtocol {
 extension SelectAnnotationExample: AnnotationInteractionDelegate {
     public func annotationManager(_ manager: AnnotationManager,
                                   didDetectTappedAnnotations annotations: [Annotation]) {
-        annotationSelected.toggle()
+        if annotationSelected || annotations.count > 0 {
```

Comment:
```suggestion
        if annotationSelected || !annotations.isEmpty {
```

Use isEmpty when possible for improved performance: https://developer.apple.com/documentation/swift/array/1688398-isempty

Reply:
Thank you!